components/layout_2020/formatting_contexts.rs

```diff
@@ -187,18 +192,26 @@ impl NonReplacedFormattingContext {
         &self,
         layout_context: &LayoutContext,
         positioning_context: &mut PositioningContext,
+        containing_block_for_children: &ContainingBlock,
         containing_block: &ContainingBlock,
```

Comment:
Just for the sake of clarity, perhaps this can be:

```suggestion
        containing_block_from_parent: &ContainingBlock,
```